### PR TITLE
Rename `tsv` format to `compact` & `bdbag` to `terra.bdbag` (#1086)

### PR DESCRIPTION
**Pull request by achave11**

### CODE REVIEW

File: lambdas/service/app.py [hunk 1]
```diff
@@ -739,8 +739,8 @@ def handle_manifest_generation_request():
     try:
         format_ = query_params['format']
     except KeyError:
-        format_ = 'tsv'
-    if format_ not in ('tsv', 'bdbag', 'full'):
+        format_ = 'compact'
+    if format_ not in ('compact', 'terra_bdbag', 'full'):
```
Comment (danielsotirhos) [inline lambdas/service/app.py]:
```suggestion
    if format_ not in ('compact', 'tsv', 'terra_bdbag', 'bdbag', 'full'):
```

File: src/azul/service/responseobjects/elastic_request_builder.py [hunk 2]
```diff
@@ -547,7 +547,7 @@ def transform_manifest(self, format_: str, filters: JSON):
             assert len(response.hits) == 0
             aggregate = response.aggregations
             manifest_config = self._generate_full_manifest_config(aggregate)
-        elif format_ == 'bdbag':
+        elif format_ == 'terra_bdbag':
```
Comment (danielsotirhos) [inline src/azul/service/responseobjects/elastic_request_builder.py]:
```suggestion
        elif format_ == 'terra_bdbag' or format_ == 'bdbag':
```

File: src/azul/service/responseobjects/hca_response_v5.py [hunk 3]
```diff
@@ -298,8 +298,8 @@ def _push_content(self) -> str:
         with MultipartUploadHandler(object_key, content_type) as multipart_upload:
             with FlushableBuffer(AWS_S3_DEFAULT_MINIMUM_PART_SIZE, multipart_upload.push) as buffer:
                 text_buffer = TextIOWrapper(buffer, encoding="utf-8", write_through=True)
-                if self.format == 'tsv':
-                    self._write_tsv(text_buffer)
+                if self.format == 'compact':
```
Comment (danielsotirhos) [inline src/azul/service/responseobjects/hca_response_v5.py]:
```suggestion
                if self.format == 'compact' or self.format == 'tsv':
```

File: src/azul/service/responseobjects/hca_response_v5.py [hunk 4]
```diff
@@ -324,14 +324,14 @@ def _push_content_single_part(self) -> str:
 
         :return: S3 object key
         """
-        if self.format == 'tsv':
+        if self.format == 'compact':
```
Comment (danielsotirhos) [inline src/azul/service/responseobjects/hca_response_v5.py]:
```suggestion
        if self.format == 'compact' or self.format == 'tsv':
```

File: src/azul/service/responseobjects/hca_response_v5.py [hunk 5]
```diff
@@ -324,14 +324,14 @@ def _push_content_single_part(self) -> str:
...
             output = StringIO()
-            self._write_tsv(output)
+            self._write_compact(output)
             return self.storage_service.put(object_key=f'manifests/{self.object_key}.tsv',
                                             data=output.getvalue().encode(),
                                             content_type='text/tab-separated-values')
-        elif self.format == 'bdbag':
-            bdbag_path = self._create_bdbag_archive()
+        elif self.format == 'terra_bdbag':
```
Comment (danielsotirhos) [inline src/azul/service/responseobjects/hca_response_v5.py]:
```suggestion
        elif self.format == 'terra_bdbag' or self.format == 'bdbag':
```

File: src/azul/service/responseobjects/hca_response_v5.py [hunk 6]
```diff
@@ -548,7 +548,7 @@ def _dss_url(self, file):
         return dss_url
 
     def return_response(self):
-        if config.disable_multipart_manifests or self.format == 'bdbag':
+        if config.disable_multipart_manifests or self.format == 'terra_bdbag':
```
Comment (danielsotirhos) [inline src/azul/service/responseobjects/hca_response_v5.py]:
```suggestion
        if config.disable_multipart_manifests or self.format == 'terra_bdbag' or self.format == 'bdbag':
```

File: src/azul/service/responseobjects/hca_response_v5.py [hunk 7]
```diff
@@ -194,7 +194,7 @@ def __init__(self, es_search, manifest_entries, mapping, format_, object_key=Non
         The constructor takes the raw response from ElasticSearch and creates
         a csv file based on the columns from the manifest_entries
         :param es_search: The Elasticsearch DSL Search object
-        :param manifest_entries: The columns that will be present in the tsv
+        :param manifest_entries: The columns that will be present in the compact
```
Comment (hannes-ucsc) [inline src/azul/service/responseobjects/hca_response_v5.py]: Search/replace error.
```suggestion
        :param manifest_entries: The columns that will be present in the manifest
```

**Comment by hannes-ucsc** [comment]
And the commit title needs to be adjusted, too.

https://github.com/DataBiosphere/azul/pull/1306#event-2663826583

**Pull request merged**